Drop the separate Boost.System link dependency, which is not needed by
the current Boost configuration.
Index: tests/Makefile
--- tests/Makefile.orig
+++ tests/Makefile
@@ -18,7 +18,6 @@ ifneq (, $(findstring mingw, $(SYS))$(fi
 endif
 
 LDLIBS = \
-	-lboost_system$(BOOST_SUFFIX) \
 	-lboost_program_options$(BOOST_SUFFIX) \
 	-lssl \
 	-lcrypto \
